src/js/dp/feedback-form.js

```diff
@@ -0,0 +1,122 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const pageURL = window.location.href;
+    const feedbackURL = "/feedback";
+
+
+    const feedbackFormContainer = document.querySelector(
```

Comment:
Could use the forms api
```suggestion
    const feedbackFormContainer = document.forms(
```